docs/howtos/Release_Process_with_PR_Rebase.md

```diff
@@ -0,0 +1,35 @@
+# Release Process with PR Rebase
+
+Rebasing your pull requests keeps your commit history clean and integrates changes from the main branch, ensuring a smoother merge.
+
+**Note:** the current release is tagged as `release-v*`. Replace `release-v*` with the actual version.
+
+#### Before creating a new local working branch:
+   - Make sure you have local versions of the Learning Equality `develop` branch and the Learning Equality `release-v*` branch.
+   - Ensure that both branches are up to date. For this guide, we'll assume they are named `develop` and `release-v*`, respectively.
+   
+Checkout the upstream develop branch:
+```
+git checkout develop
+```
+After making changes to the code and committing them locally, push your working branch to your fork on GitHub:
+
+```
+git push origin develop
+```
+
```

Comment:
**Lines 11-20:**

Most of people don't work directly on `develop` but rather on a feature branch started from the `develop`. So I think putting it this way would be rather confusing. Could you use the language of the feature branch suggested in https://github.com/learningequality/kolibri/issues/11361?

> Locally, checkout your feature branch

If you'd like to add a bit of guidance to make clear that it's a branch going from `develop`, you could for example say

> Locally, checkout your feature branch (that you previously created off the `develop`)

Also, I think' there's no need to be explaining `git push origin develop` here.